src/DynamoCore/Library/LibraryServices.cs

```diff
@@ -518,8 +520,9 @@ private static bool CanbeResolvedTo(ICollection<string> partialName, ICollection
         /// <summary>
         ///     Import a library (if it hasn't been imported yet).
         /// </summary>
-        /// <param name="library"></param>
-        internal bool ImportLibrary(string library)
+        /// <param name="library">The library to be loaded</param>
+        /// <param name="isLocalLib">Indicates if the library has been locally imported</param>
```

Comment:
I'm confused by this- when is a library not locally imported? What do you mean by locally imported?

Reply:
A library is considered "locally imported" when it was imported by using the "File | ImportLibrary..." command. There is a specific location where this occurs and the isLocalLib flag is only set there.

Reply:
I am looking for the definition of locallib to be in the code - your summary doc is self referencing.

for example, a package that imports a library seems like a locally imported dll to me.

Reply:
Renamed parameter name and added many more comments to make it as clear as possible